Eatery Blue/Eatery Blue/UI/EateryScreen/EateryModelController.swift

```diff
@@ -131,7 +145,7 @@ class EateryModelController: EateryViewController {
         categoryViews.removeAll()
     }
 
-    private func setUpMenuFromState() {
+    private func addMenuFromState() {
```

Comment:
I checked other swift files and saw you usually say "setUp....". Why change it to "add" here?

Reply:
This is definitely something to document, In general I use `setUp` for functions that ought to be called once, `update` for functions that ought to be called multiple times, and `add` for functions that add stuff to a stack view

Reply:
Ahhh ok that clarifies it